CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to Bicep
+We are very happy to accept community contributions to Bicep, whether those are [Pull Requests](#pull-requests), [Example Files](#example-files), [Feature Suggestions](#feature-suggestions) or [Bug Reports](#bug-reports)! Please note that by participating in this project, you agree to abide by the [Code of Conduct](./CODE_OF_CONDUCT.md), as well as the terms of the [CLA](#cla).
+
+## Getting Started
+* If you haven't already, you will need [dotnet](https://dotnet.microsoft.com/download) and [node + npm](https://nodejs.org/en/download/) installed locally to build and run this project.
```

Comment:
Should probably specify the versions of dotnet and node. For dotnet, we can point to global.json as the minimum (it will use a higher non-preview version). For node, we can probably just say 10.x or something like that.

Reply:
Updated. Linking to global.json feels a bit too complicated for anyone not familiar with global.json, so I've just put "dotnet 3.1 (or later)". AFAIK you get a pretty clear message if you've got the wrong version of dotnet installed, so I don't think this would cause much of a problem, and I want to optimize for people unfamiliar with dotnet.

Reply:
Was hoping to avoid stale info here if we ever change global.json. Also, the min version is 3.1.301 right now.

---
In reply to: [496854769](https://github.com/Azure/bicep/pull/567#discussion_r496854769) [](ancestors = 496854769)